src/components/custom/js/functions.js

```diff
@@ -334,6 +334,22 @@ export function getStatusColor(status) {
 
 }
 
+export const datasetIs = {
```

Comment:
@libpitt Instead of going off of the creation action please utilize the `dataset_category` property instead. That gets set to either `primary`, `component`, `codcc-processed` (this is the processed dataset category), or `lab-processed`.